website/docs/d/is_dedicated_host_disk.html.markdown

```diff
@@ -13,9 +13,9 @@ Retrieve the dedicated host disk. For more information, about dedicated host dis
 ## Example usage
 
 ```terraform
-data "ibm_is_dedicated_host_disk" "is_dedicated_host_disk" {
-	dedicated_host = "dedicatedhost id"
-	disk = "id"
+data "ibm_is_dedicated_host_disk" "example" {
+	dedicated_host = "ibm_is_dedicated_host.example.id"
+	disk = "data.ibm_is_dedicated_host_disks.test1.disks.0.id"
```

Comment:
Doesn't look right? 
Please double check all docs for `"ibm_is_dedicated_host.example.id"`  should be `bm_is_dedicated_host.example.id`  no `""` 

also incorrect `"data.ibm_is_dedicated_host_disks.test1.disks.0.id"` no example ?